### DISCUSSION TITLE
DesktopAcrylicBackdrop type missing in Uno.WinUI ?

### DISCUSSION THREAD

**Discussion by VincentH-Net** [unoplatform/uno#13469] [Q&A] [answered]
Does anyone know why the latest stable `Uno.WinUI` (4.9.45) does not have the type `Microsoft.UI.Xaml.Media.DesktopAcrylicBackdrop`?

The latest stable `Microsoft.WindowsAppSDK` (1.4.230822000) does have this type, and contains several theme resources of this type.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. It's not updated yet, uno is still based on 1.2: https://github.com/unoplatform/uno/issues/13470

**Reply by VincentH-Net** [reply]
Thanks for clarifying - I will revisit when #13470 is closed
